projects/homework 4/binary_representation/main.c

```diff
@@ -0,0 +1,105 @@
+#include <stdio.h>
+#include <math.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+#define SIZE 4
```

Comment:
Всякие штуки типа арифметического переполнения (которое тут может быть полезно при переводе обратно) будут правильно работать только если размер числа равен размеру int в битах, так что должно быть вот так:
```suggestion
#define SIZE (sizeof(int) * 8)
```